main.go

```diff
@@ -327,7 +327,6 @@ func loop() {
 
 	extraOrigins := []string{
 		"https://create.arduino.cc",
-		"https://create-dev.arduino.cc", "https://create-intel.arduino.cc",
```

Comment:
@endorama I removed these origins since they are old and rarely used, plus it's not good to have them hardcoded.
It's possible to add new origins using the `origins` filed in the `config.ini` file.
Just add your origins and separate them with ", "

Reply:
@umbynos is the description of the PR correct after the last changes?